auto3dseg/algorithm_templates/dints/scripts/train.py

```diff
@@ -260,47 +354,63 @@ def run(config_file: Optional[Union[str, Sequence[str]]] = None, **override):
             dist.get_rank()]
     logger.debug(f"val_files: {len(val_files)}")
 
-    train_cache_rate = float(parser.get_parsed_content("training#train_cache_rate"))
+    train_cache_rate = float(parser.get_parsed_content("train_cache_rate"))
     validate_cache_rate = float(
-        parser.get_parsed_content("training#validate_cache_rate"))
+        parser.get_parsed_content("validate_cache_rate"))
```

Comment:
Similarly, it should have "training#" before "validate_cache_rate"

Reply:
I have fixed this line.